pkg/daemon/daemon.go

```diff
@@ -949,10 +949,11 @@ func (p *ptpProcess) processPTPMetrics(output string) {
 		logEntry := synce.ParseLog(output)
 		p.ProcessSynceEvents(logEntry)
 
-	} else if p.name == ts2phcProcessName && (strings.Contains(output, NMEASourceDisabledIndicator) ||
-		strings.Contains(output, InvalidMasterTimestampIndicator) ||
-		(strings.Contains(output, NMEASourceDisabledIndicator2) &&
-			(!leap.LeapMgr.IsLeapInWindow(time.Now().UTC(), -2*time.Second, time.Second)))) { //TODO identify which interface lost nmea or 1pps
+	} else if p.name == ts2phcProcessName &&
+		!leap.LeapMgr.IsLeapInWindow(time.Now().UTC(), -2*time.Second, time.Second) &&
+		(strings.Contains(output, NMEASourceDisabledIndicator) ||
+			strings.Contains(output, InvalidMasterTimestampIndicator) ||
+			strings.Contains(output, NMEASourceDisabledIndicator2)) { //TODO identify which interface lost nmea or 1pps
```

Comment:
isn't nmea  lost indicator always for GM interface ? GM interface will  give the nmea interface , until we support two GNSS

Reply:
That's right, only for GM interface. I only added a condition for declaring "nmea string lost", that we are not in the leap event right now